projects/demo/src/modules/app/stackblitz/stackblitz.service.ts

```diff
@@ -106,31 +76,15 @@ export class TuiStackblitzService implements TuiCodeEditor {
     }
 
     private async getBaseAngularProjectFiles(): Promise<Project['files']> {
-        const {tsconfig, angularJson, mainTs, polyfills, indexHtml, styles, appModuleTs} =
+        const {tsconfig, angularJson, mainTs, indexHtml, globalStyles} =
             await AbstractTuiStackblitzResourcesLoader.getProjectFiles();
 
         return {
             'tsconfig.json': tsconfig,
             'angular.json': angularJson,
             'src/main.ts': mainTs,
-            'src/polyfills.ts': polyfills,
             'src/index.html': indexHtml,
-            'src/styles.less': styles,
-            [appPrefix`app.module.ts`]: appModuleTs.toString(),
-        };
-    }
-
-    /** Some stackblitz hacks */
-    private async getStackblitzOnlyFiles(
-        additionalModules: Array<[fileName: string, parsedFile: TsFileModuleParser]> = [],
-    ): Promise<Project['files']> {
-        const {stackblitzReadMe} =
-            await AbstractTuiStackblitzResourcesLoader.getReadMeFiles();
-
-        return {
-            [stackblitzPrefix`README.md`]: stackblitzReadMe,
-            [stackblitzPrefix`all-taiga-modules.ts`]:
-                await getAllTaigaUIModulesFile(additionalModules),
+            'src/global_styles.less': globalStyles,
```

Comment:
Naming is changed to be similar to the most revent version of angular stackblitz starter

https://github.com/stackblitz/starters/blob/main/angular/src/global_styles.css